physioqc/interfaces/visualizations.py

```diff
@@ -0,0 +1,197 @@
+from typing import List, Literal, Tuple
+
+import matplotlib.pyplot as plt
+import numpy as np
+import peakdet as pk
+
+from ..metrics import multimodal
+
+
+def check_create_figure(
+    ax: plt.axes = None, figsize=(10, 5)
+) -> Tuple[plt.figure, plt.axes]:
+    """Helper function to check if a new figure should be created or to further
+    plot on axes.
+
+    Parameters
+    ----------
+    ax : plt.axes, optional
+        Check is axes object is present, if not creates new figure, by default None
+    figsize : tuple, optional
+        Size of the new figure (if created), by default (10, 5)
+
+    Returns
+    -------
+    Tuple[plt.figure, plt.axes]
+        Figure and axes objects of the plot.
+    """
+
+    if ax is None:
+        fig, ax = plt.subplots(1, 1, figsize=figsize)
+    else:
+        fig = None
+
+    return fig, ax
+
+
+def plot_raw_data(phys: pk.Physio, ax: plt.axes = None) -> Tuple[plt.figure, plt.axes]:
+    """Plots the raw data using peakdet.
+    Parameters
+    ----------
+    phys : pk.Physio
+        The data object, used for plotting.
+    ax : plt.axes, optional
+        axes object to use for plotting, by default None
+
+    Returns
+    -------
+    Tuple[plt.figure, plt.axes]
+        Figure and axes objects of the plot.
+    """
+    fig, ax = check_create_figure(ax, figsize=(7, 5))
+
+    ax = pk.plot_physio(phys, ax=ax)
+
+    return fig, ax
+
+
+def plot_average_peak(
+    phys: pk.Physio,
+    window: List = [-3, 3],
+    target_fs: float = 1000.0,
+    peak_dist: float = 1.0,
+    peak_thr: float = 0.1,
+    plot_mode: Literal["traces", "ci", "auto"] = "auto",
+    ax: plt.axes = None,
+) -> Tuple[plt.figure, plt.axes]:
+    """_summary_
+
+    Parameters
+    ----------
+    phys : pk.Physio
+        data object, used for plotting.
+    window : List, optional
+        window size around the peak in s, by default [-3, 3]
+    target_fs : float, optional
+        sampling rate for plotting and peak detection, by default 1000.0
```

Comment:
It seems like target_fs is only used if the sampling rate of the data is greater than 1000, so it'd be helpful to mention that here